c/tskit/tables.h

```diff
@@ -686,6 +686,15 @@ reference them. */
 #define TSK_SIMPLIFY_FILTER_POPULATIONS (1 << 1)
 /** Remove individuals from the output if there are no nodes that reference them.*/
 #define TSK_SIMPLIFY_FILTER_INDIVIDUALS (1 << 2)
+/** Do not remove nodes from the output if there are no edges that reference
+them. Note that this flag is negated compare to other filtering options because
+the default behaviour is to filter unreferenced nodes.
```

Comment:
```suggestion
/** Do not remove from the output if there are no edges that reference
them, and do not reorder nodes so that the samples are nodes 0 to num_samples - 1.
Note that this flag is negated compared to other filtering options because
the default behaviour is to filter unreferenced nodes and reorder to put samples first.
```